README.md

```diff
@@ -1,14 +1,13 @@
-## Overview
-OptiRTC-API-examples provides examples of using open-source technologies to interact with OptiRTC Public HTTP APIs. To assess all possible API calls for your use case, please refer to the documentation at the https://*.optirtc.com/Help subdomain of the OptiRTC API you are interested in using (e.g. https://public.optirtc.com/Help).
+# Overview
 
-To receive messages about updates to these examples, log in to Github and "Watch" this repository.
+OptiRTC-API-examples provides examples of using open-source technologies to interact with OptiRTC Public HTTP APIs. To assess all possible API calls for your use case, please refer to the documentation at the `https://*.optirtc.com/Help` subdomain of the OptiRTC API you are interested in using (e.g. https://public.optirtc.com/Help).
```

Comment:
OptiRTC-API-examples provides examples of using open-source technologies to interact with OptiRTC Web APIs. For API documentation, please see: https://optirtc.com/resources#web-apis.

Reply:
Added.